components/drops/useDrops.ts

```diff
@@ -60,7 +60,7 @@ const getFormattedDropItem = async (collection, drop: DropItem) => {
     collection: collection,
     minted: Math.min(count, chainMax),
     max: chainMax,
-    dropStartTime: FUTURE_DROP_DATE,
+    dropStartTime: count >= 2 ? Date.now() - 1e10 : FUTURE_DROP_DATE, // this is a bad hack to make the drop appear as "live" in the UI
```

Comment:
The hack does not work for now because of the changed count : )
<img width="292" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/40f2afa3-c5e2-49ab-ae0b-50ec1fa9dcfb">